buildSrc/src/main/groovy/org/apache/beam/gradle/DependenciesPlugin.groovy

```diff
@@ -0,0 +1,269 @@
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class DependenciesPlugin implements Plugin<Project> {
+  // These versions are defined here because they represent
+  // a dependency version which should match across multiple
+  // Maven artifacts.
+  static def activemq_version = "5.14.5"
+  static def autovalue_version = "1.7.4"
+  static def aws_java_sdk_version = "1.11.718"
+  static def aws_java_sdk2_version = "2.13.54"
+  static def cassandra_driver_version = "3.10.2"
+  static def checkerframework_version = "3.10.0"
+  static def classgraph_version = "4.8.65"
+  static def google_clients_version = "1.30.10"
+  static def google_cloud_bigdataoss_version = "2.1.6"
+  static def google_cloud_pubsublite_version = "0.7.0"
+  static def google_code_gson_version = "2.8.6"
+  static def google_oauth_clients_version = "1.31.0"
+  // Try to keep grpc_version consistent with gRPC version in google_cloud_platform_libraries_bom
+  static def grpc_version = "1.35.0"
+  static def guava_version = "30.1-jre"
+  static def hadoop_version = "2.10.1"
+  static def hamcrest_version = "2.1"
+  static def influxdb_version = "2.19"
+  static def httpclient_version = "4.5.10"
+  static def httpcore_version = "4.4.12"
+  static def jackson_version = "2.12.1"
+  static def jaxb_api_version = "2.3.3"
+  static def jsr305_version = "3.0.2"
+  static def kafka_version = "2.4.1"
+  static def nemo_version = "0.1"
+  static def netty_version = "4.1.52.Final"
+  static def postgres_version = "42.2.16"
+  static def powermock_version = "2.0.9"
+  static def protobuf_version = "3.12.0"
+  static def quickcheck_version = "0.8"
+  static def slf4j_version = "1.7.30"
+  static def spark_version = "2.4.7"
+  static def spotbugs_version = "4.0.6"
+  static def testcontainers_version = "1.15.1"
+
+  @Override
+  void apply(Project project) {
+    // Define and export a map dependencies shared across multiple sub-projects.
+    //
+    // Example usage:
+    // configuration {
+    //   compile library.java.avro
+    //   testCompile library.java.junit
+    // }
+
+
+    // A map of maps containing common libraries used per language. To use:
+    // dependencies {
+    //   compile library.java.slf4j_api
+    // }
```

Comment:
It looks like we unintentionally had these two different examples in BeamModulePlugin.groovy. I think the first one is just wrong, I can find any examples of a `configuration {` block. Please combine these into one, something like:
 ```suggestion
    // Define and export a map of maps containing common library dependencies
    // per language to share across multiple sub-projects.
    //
    // Example usage:
    // dependencies {
    //   compile library.java.slf4j_api
    //   testCompile library.java.junit
    // }
```

Reply:
Fixed